data/human/outfits.txt

```diff
@@ -399,6 +409,15 @@ outfit "Local Map"
 	"map" 12
 	description "This data chip contains complete information on the twelve star systems closest to this one: planets, ports, governments, trade prices, available services, etc. You can get all the same information just by exploring those systems yourself, but having a map can save you from making wrong turns if you are trying to travel through new territory to reach a certain system quickly."
 
+outfit "Slave"
+	category "Special"
+	cost 25000
+	thumbnail "outfit/slave"
+	"required crew" -1
+	"outfit space" -1
+	"atrocity" 1
+	"unplunderable" 1
+	description "Slaves are highly illegal but can be a great asset if you are going on a long journey into the middle of nowhere. They take the place of a crew member, and you don't have to pay them a daily wage. They also come with a small bed and space to sleep in."
```

Comment:
I worked out what bothers me about this description - it describes the slave as an object, a thing. You're not treating the human with anywhere near the amount if sympathy they deserve. The description needs to focus on the cruelty of the person trying to sell you another person, and on the inhumanity of any captain who buys one. This current description makes a human out to be a piece of machinery, something with pros and cons to installing on your ship; it's dehumanizing.